README.md

```diff
@@ -77,6 +77,32 @@ Custom headers allowed by CORS, Access-Control-Allow-Headers.
 
 Valid time for pre-flight, Access-Control-Max-Age.
 
+### laravel_https_enabled
+
+Enable https
+
+Default is `false`
+
+### laravel_https_cert_path
+
+SSL certificate path
+
+### laravel_https_key_path
+
+SSL certificate private key path
+
+### laravel_https_cert_data
+
+SSL certificate
+
+### laravel_https_digicert_data
```

Comment:
Digicert is a company providing SSL certificates. Let's make this more generic and call it `laravel_https_chain_data`. Chain as in the certificate chain, which can include intermediate and root certificates.